<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>205</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5,5,5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="logoLabel">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/logo);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="loginTitleLabel">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;贝壳找房&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username">
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>admin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>用户名</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>123456</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>类别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type">
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>管理员</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>普通用户</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5,5">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="OkBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>205</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
